<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrainFlowSetupWidget</class>
 <widget class="QWidget" name="BrainFlowSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>Serial Port</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="serialPort"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>MAC Address</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="macAddress"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="text">
        <string>IP Address</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ipAddress"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="text">
        <string>IP Port</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ipPort"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="text">
        <string>IP Protocol</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ipProtocol"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="text">
        <string>Other Info</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="otherInfo"/>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="text">
        <string>Serial Number</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="serialNumber"/>
     </item>
     <item row="8" column="0" colspan="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0">
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="text">
        <string>Streamer Params</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="streamerParams">
       <property name="text">
        <string>file://results.csv:w</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="0">
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="text">
        <string>Board Id</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="boardId"/>
     </item>
     <item row="12" column="0" colspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="13" column="0">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>Data to Display</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QComboBox" name="dataType"/>
     </item>
     <item row="17" column="0" colspan="2">
      <widget class="QPushButton" name="prepareSession">
       <property name="text">
        <string>Submit Params and Prepare Session</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0" colspan="2">
      <widget class="QPushButton" name="releaseSession">
       <property name="text">
        <string>Release Session</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="file"/>
     </item>
     <item row="16" column="0" colspan="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
